src/main/webapp/assets/css/feedStyle.css

```diff
@@ -15,8 +15,13 @@
   padding: 0 0 0 15px;
 }
 
+#find-events-title:focus {
+  color: var(--dark-green);
+  outline: none;
+}
+
 /*
- * Create Posts
+ * Create Posts (Button + Modal).
  */
 #create-post-button {
   background-color: #303030;
```

Comment:
Since #303030 is a common hex I see here, creating a variable for it would be helpful! :)